_releases/2022-04-01-march-2022.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+category: releases
+title: March 2022 Release Notes
+filename-custom: march-2022
+last_modified_at: April 1, 2022
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements
+
+* **Gem Updates:** We’ve updated gems as needed for a secure environment.
+* **Legacy Code Removal:** We've removed unused code associated with a legacy feature, Search Consumer.
+* **ElasticSearch Upgrade:** We've made progress on updating our system to use ElasticSearch 7.
```

Comment:
Please edit to make general, and not mention specific version number thanks

Reply:
Updated